xl/playlist.py

```diff
@@ -2170,7 +2175,13 @@ def load_names(self):
                 except Exception:
                     logger.exception("Failed loading playlist: %r", path)
                 else:
-                    existing.append(pl.name)
+                    name = pl.name
+
+                    # Make sure name is unicode
```

Comment:
(As above.)

Reply:
Same as above; the conversion does seem to be necessary when re-starting exaile after renaming a playlist. But not on subsequent restarts...